dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -51,6 +51,10 @@ SolverUnstableZMode         =             discrete events are unstable during th
 SolverUnbalanced            =             the number of algebraic/differential variables is different from the number of algebraic/differential equations in the simulated problem
 WrongLinearSolverChoice     =             the linear solver name provided is not valid
 SolverMissingParam          =             no value found for mandatory solver parameter '%1%' (parameter set '%2%', parameter file '%3%')
+SolverJacobianWithNulColumn =             jacobian has nul column %1%: equation '%2%' is ill-formed
```

Comment:
You will also need to update french dico

Reply:
OK